### ISSUE TITLE
训练模型后显示成功，没报错，但是没生成模型

### ISSUE THREAD

**Issue by Wangxiaobin123** [PaddlePaddle/PaddleNLP#5273]
### 请提出你的问题

运行：
python -u -m paddle.distributed.launch finetune.py \                 
--train_path /Users/wangshengbin/shengbin/paddle-sentiments/pnlp/data/train.json \
  --dev_path /Users/wangshengbin/shengbin/paddle-sentiments/pnlp/data/dev.json \
  --save_dir /Users/wangshengbin/shengbin/paddle-sentiments/pnlp/checkpoint/ \
  --learning_rate 1e-5 \
  --batch_size 16 \
  --max_seq_len 512 \
  --num_epochs 3 \
  --model uie-senta-base \
  --seed 1000 \
  --logging_steps 10 \
  --valid_steps 100 \
  --device cpu
运行的结果：
[2023-03-16 18:43:58,115] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load 'uie-senta-base'.
[2023-03-16 18:43:58,116] [    INFO] - Already cached /Users/wangshengbin/.paddlenlp/models/uie-senta-base/uie-senta-base/ernie_3.0_base_zh_vocab.txt
[2023-03-16 18:43:58,155] [    INFO] - tokenizer config file saved in /Users/wangshengbin/.paddlenlp/models/uie-senta-base/uie-senta-base/tokenizer_config.json
[2023-03-16 18:43:58,155] [    INFO] - Special tokens file saved in /Users/wangshengbin/.paddlenlp/models/uie-senta-base/uie-senta-base/special_tokens_map.json
[2023-03-16 18:43:58,159] [    INFO] - Model config ErnieConfig {
  "attention_probs_dropout_prob": 0.1,
  "enable_recompute": false,
  "fuse": false,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 2048,
  "model_type": "ernie",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "paddlenlp_version": null,
  "pool_act": "tanh",
  "task_id": 0,
  "task_type_vocab_size": 3,
  "type_vocab_size": 4,
  "use_task_id": true,
  "vocab_size": 40000
}

[2023-03-16 18:44:19,416] [    INFO] - All model checkpoint weights were used when initializing UIE.

[2023-03-16 18:44:19,423] [    INFO] - All the weights of UIE were initialized from the model checkpoint at uie-senta-base.
If your task is similar to the task the model of the checkpoint was trained on, you can already use UIE for predictions without further training.
LAUNCH INFO 2023-03-16 18:50:54,524 Pod completed
[2023-03-16 18:50:54,524] [    INFO] controller.py:98 - Pod completed
LAUNCH INFO 2023-03-16 18:50:54,553 Exit code 0
[2023-03-16 18:50:54,553] [    INFO] controller.py:141 - Exit code 0

checkpoint 目录是空的

**Comment by yyh59098** [comment]
我也遇到和你同样的问题，这个您解决了吗？

**Comment by Wangxiaobin123** [comment]
> 我也遇到和你同样的问题，这个您解决了吗？

还没有，不知道什么原因

**Comment by yyh59098** [comment]
好吧

**Comment by 526872163** [comment]
调低 batch_size,爆显存了，
训练并没有进行

**Comment by Wangxiaobin123** [comment]
> 调低 batch_size,爆显存了， 训练并没有进行
好的 可以了 从日志好像没有看到相关信息，请教下是哪里有体现吗

**Comment by Wangxiaobin123** [comment]
> 好吧

调低 batch_size 就可以了
